Source/Core/DolphinQt/Achievements/AchievementsWindow.cpp

```diff
@@ -36,7 +36,7 @@ AchievementsWindow::AchievementsWindow(QWidget* parent) : QDialog(parent)
         });
       });
   connect(&Settings::Instance(), &Settings::EmulationStateChanged, this,
-          [this] { m_settings_widget->UpdateData(); });
+          [this] { m_settings_widget->UpdateData(0); });
```

Comment:
Using a raw 0 feels a bit ugly here. Could we use `RC_OK`? Or would you rather not include that header?

Reply:
You're now using `RC_OK` without including the required header. (It's currently relying on an indirect include.)